scripts/vet.sh

```diff
@@ -167,4 +168,10 @@ XXXXX PleaseIgnoreUnused'
   popd
 done
 
+# - Collection of revive linter analysis checks
+REV_OUT="$(mktemp)"
+revive -formatter unix ./... >"${REV_OUT}" || true
+
+(noret_grep -v "[unused-parameter]" "${REV_OUT}" | not grep -v "\.pb\.go:") || true
```

Comment:
Now this doesn't show us the other errors.

Maybe we can just make revive skip that check?  I assume that's possible?

Reply:
After switching to `plain` formatter, it works